lib/execution_engine2/db/MongoUtil.py

```diff
@@ -155,41 +156,28 @@ def _get_collection(
 
         return pymongo_client, mongoengine_client
 
-    @contextmanager
-    def pymongo_client(self, mongo_collection):
-        """
-        Instantiates a mongo client to be used as a context manager
-        Closes the connection at the end
-        :return:
-        """
-        self.mongo_collection = mongo_collection
-        yield self.pymongoc
-
     def get_workspace_jobs(self, workspace_id):
         with self.mongo_engine_connection():
             job_ids = [str(job.id) for job in Job.objects(wsid=workspace_id)]
             return job_ids
 
     def get_job_log_pymongo(self, job_id: str = None):
 
```

Comment:
A lot of this is mostly whitespace changes - it'll be easier to read with whitespace off